### PR TITLE
Proposed update of logging on section

### PR DESCRIPTION
**Pull request by Sparrow0hawk**
This PR revamps our current logging on documentation under the Getting Started section.

It separates out different aspects of logging on and attempts to provide more inline links to steer the user to their desired walkthrough (based on OS and location).

Changes summary: 
- Moved to a multi-page set up (separating connecting from campus or off campus)
- Added a section specifically about connecting from campus with updated guidance 
- Integrated @jhodrien section on fingerprint details as a separate section
- Separated out the section on X11 forwarding
- Slight adjustments to the wording at various walkthrough steps

### CODE REVIEW

File: book/getting_started/logon/logon-campus.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+```bash
+# to log into ARC4
+$ ssh exuser@arc4
+
+# to log into ARC3
+$ ssh exuser@arc3
+```
+
+The first time you SSH to either ARC4 or ARC3 you will be prompted to confirm the fingerprint of the host you're connecting to. You can check the unique fingerprints for ARC3 and ARC4 in the [fingerprint details](#fingerprint-details) below.
```
Comment (lukeconibear) [inline book/getting_started/logon/logon-campus.md:17]:
```suggestion
The first time you SSH to either ARC4 or ARC3 you will be prompted to confirm the fingerprint of the host you're connecting to. You can check the unique fingerprints for ARC3 and ARC4 in the [fingerprint details](fingerprint-details) below.
```

File: book/getting_started/logon/logon-campus.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+ARC3:
+
+```
+The authenticity of host 'arc3 (129.11.26.153)' can't be established.
+ECDSA key fingerprint is SHA256:cOA3zaJLawry0uVQ3VPveyxcrUJ+/8hRcr59xSbYRNI.
+ECDSA key fingerprint is MD5:c2:b1:9a:c1:44:aa:ba:a1:db:c6:b7:d4:48:9d:74:89.
+Are you sure you want to continue connecting (yes/no)?
+```
+
+You should confirm the key fingerprints match from fingerprints shown in the [fingerprint details](#fingerprint-details) section and type `yes` and press Enter to proceed.
```
Comment (lukeconibear) [inline book/getting_started/logon/logon-campus.md:35]:
```suggestion
You should confirm the key fingerprints match from fingerprints shown in the [fingerprint details](fingerprint-details) section and type `yes` and press Enter to proceed.
```

**Review by lukeconibear — CHANGES_REQUESTED** [review]
Looks good to me. Very minor typo for internal links to fingerprint-details page.

File: book/getting_started/logon/logon-campus.md (hunk 1 above)
Comment (jhodrien) [reply book/getting_started/logon/logon-campus.md:17]: Drop the word below at the end of that sentence, since it isn't anymore.

**Review by lukeconibear — APPROVED** [review]

File: book/getting_started/logon/logon-off-campus.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+| --------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![You will immediately be prompted for your password to connect to the remote-access server](../../assets/img/logon/offcampus/mobaXTerm6.png) |
+
+| 7. You will immediately be prompted for to enter your [Duo two-factor code](https://it.leeds.ac.uk/it?id=kb_article&sysparm_article=KB0014641) which can be found on your Duo App |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![You will immediately be prompted for to enter your Duo two-factor login code which can be found on your Duo App](../../assets/img/logon/offcampus/mobaXTerm6-5.png)             |
+
+| 8. After it accepts your password it will request your password again, this time to log into ARC4 (This may not happen automatically and you may need to double click on session name that appears on the right hand User sessions section) |
```
Comment (NGRhodes) [inline book/getting_started/logon/logon-off-campus.md:158]: on the right hand User sessions section

left ?

**Review by NGRhodes — CHANGES_REQUESTED** [review]
Generally fine, a few typos.
I THINK all the instructions make sense, I have not tried them all out (just the linux/mac ones).

**Review by NGRhodes — APPROVED** [review]

**Pull request merged**